_common.scss

```diff
@@ -56,4 +55,8 @@
         padding-bottom: 0.5rem;
         font-size: 0.875rem;
     }
+    th.mat-header-cell, td.mat-cell, td.mat-footer-cell, th.mat-header-cell:first-of-type, td.mat-cell:first-of-type, td.mat-footer-cell:first-of-type {
+        padding-left: 1rem;
+        padding-right: 1rem;
```

Comment:
Most of the time only vertical space should be doing the `rem` unit. The left and right padding should be doing a regular `px` unit. This is to ensure that when the system font size is increased to 2x, everything still reads fine.